sigh/lib/sigh/resign.rb

```diff
@@ -33,6 +33,7 @@ def resign(ipa, signing_identity, provisioning_profiles, entitlements, version,
       bundle_version = "--bundle-version #{bundle_version}" if bundle_version
       verbose = "-v" if $verbose
       bundle_id = "-b '#{new_bundle_id}'" if new_bundle_id
+      specific_keychain = "--keychain-path '#{keychain_path}'" if keychain_path
```

Comment:
Should `keychain_path` be shell escaped with `shellescape` in case it has spaces?